homeassistant/components/dsmr/sensor.py

```diff
@@ -234,6 +235,18 @@ def unit_of_measurement(self):
         """Return the unit of measurement of this entity, if any."""
         return self.get_dsmr_object_attr("unit")
 
+    @property
+    def device_state_attributes(self):
+        """Return the devices' state attributes."""
+        # Make sure telegram contains the identifier
+        if self._identifier not in self.telegram:
+            return None
+
+        attrs = {
+            "equipment_identifier": getattr(self.telegram[self._identifier], 'value', None)
```

Comment:
Is there no unique stable identifier we could use to implement the `unique_id` property? That's the best way to identify entities.

https://developers.home-assistant.io/docs/en/entity_index.html#generic-properties

https://developers.home-assistant.io/docs/en/entity_registry_index.html#unique-id-requirements

Reply:
Yep, I just realized i need to set unique_id property. Updating MR in a minute.